.nvmrc

```diff
@@ -1 +1 @@
-v8.2.1
+v8.11.2
```

Comment:
Updating to match
http://beta.webdriver.io/docs/gettingstarted.html#run-your-test-file